charts/pulsar/templates/pulsar-manager-deployment.yaml

```diff
@@ -58,7 +58,7 @@ spec:
 {{ toYaml .Values.pulsar_manager.resources | indent 12 }}
         {{- end }}
           ports:
-          - containerPort: {{ .Values.pulsar_manager.port }}
+          - containerPort: {{ .Values.pulsar_manager.service.port }}
```

Comment:
I assume we should use `targetPort` here, as the `targetPort` under service is for pods.

Reply:
Yep you're correct! Reading this here: https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service
```yaml
apiVersion: v1
kind: Service
metadata:
  name: my-service
spec:
  selector:
    app: MyApp
  ports:
    - protocol: TCP
      port: 80
      targetPort: 9376
```

>This specification creates a new Service object named “my-service”, which targets TCP port 9376 on any Pod with the app=MyApp label.